package.json

```diff
@@ -32,6 +32,7 @@
     "mBaaS"
   ],
   "scripts": {
+    "lint": "eslint .",
```

Comment:
I was thinking this would be the end result and skip the intermediate changes, but if this is the preferred method, NP. I will use your second proposal for the quick fix for now.

However, I would like to know your thought process -- the intermediate changes between seem subjective (adding more work when we know the end result we want is the one I proposed). Is the idea you want consistency here and change everything at once when we actually decide to deprecate Grunt in favour of NPM scripts?